docs/optimizations/CheckpointFiles.md

```diff
@@ -35,10 +36,7 @@ To make the checkpointing work, the `runtime` section specifies `checkpointFile:
 version 1.0
 
 workflow count_wf {
-  # Count to 2100 at 1/second => 35 minutes to complete, but
-  # luckily the state can be checkpointed every 10 minutes in
-  # case of preemption: 
-  call count { input: count_to = 2100 }
```

Comment:
This was probably the cause of your rebase issues. I modified this example in https://github.com/broadinstitute/cromwell/pull/6164 so that it actually lasts long enough to generate a checkpoint (over 10 minutes). Can we make sure that edit is not reverted in this PR?

Reply:
Yeah I was wondering about that -- it looks like there is some lack of sync between what got built into the RTD docs and the develop branch? Will amend this PR but maybe worth taking a look at the RTD setup.

Reply:
The RTD have separate publications for `develop` (the `develop` branch) vs `stable` (the `master` branch) so depending on when the last release was there's a chance the `stable` docs could be a few weeks behind the develop branch